keras/preprocessing/image.py

```diff
@@ -540,6 +541,9 @@ def random_transform(self, x):
         img_col_axis = self.col_axis - 1
         img_channel_axis = self.channel_axis - 1
 
+        if seed is not None:
+            np.random.seed(seed)
+
```

Comment:
This conditional is not necessary. You can call np.random.seed(None) unconditionally.
https://docs.scipy.org/doc/numpy/reference/generated/numpy.random.seed.html

Reply:
Correction - it is necessary. Without it np.random.seed(None) will re-seed. I can see the access to /dev/urandom (Numpy docs are slightly unclear).